pyproject.toml

```diff
@@ -44,3 +128,19 @@ select = ["ALL"]
 
 [tool.ruff.lint.pylint]
 max-args = 10
+
+[tool.cibuildwheel]
+build-verbosity = 1
+skip = ["cp36-*", "cp37-*", "pp*", "*musllinux*"]
+
+# Platform-specific configurations
+[tool.cibuildwheel.linux]
+archs = ["x86_64", "aarch64"]
+# environment = { SKBUILD_BUILD_OPTIONS = "--", CXXFLAGS = "-D_GLIBCXX_USE_CXX11_ABI=0", AUDITWHEEL_PLAT = "manylinux2014_x86_64" }
```

Comment:
Are these configurations useful?

Reply:
Yeah! Some of this cruft :) I'll clean it up. I sent the draft PR once it started working.